doc/ref/chap-type-method.md

```diff
@@ -649,8 +663,6 @@ setvar mydict['k'] += 3  # TODO: default value of 0
 
 ### Dict/clear()
 
-TODO:
-
 Remove all entries from the Dict:
 
```

Comment:
Let's also improve this entry, with something like

```
var d = {key: 'val'}
= len(d)  # => 1
call d->clear()
= len(d)  # => 0
```

So the user has something to verify

It makes the examples more readable